patrickfweston/workbench_reviewer
# #3: Permissions

To test:
1. Uninstall the module: `drush pm-uninstall workbench_reviewer -y`
1. Install the module: `drush en workbench_reviewer -y`
1. Create a new role
1. Grant the new role enough permissions so they can edit content, etc.
1. Under "Workbench Reviewer", grant them `Be assigned Workbench Reviewer` and `Assign Workbench Reviewer`.
1. Create a user of that role and log in as that user.
1. Create a node. You should see the workbench reviewer field.
1. Try entering `admin` and your user name. Both should be allowable options.
1. Change the permission for that role by unchecking `Be assigned Workbench Reviewer`.
1. Your username should no longer be an option in the box.
1. Change the permission for that role by unchecking `Assign Workbench Reviewer`.
1. You should no longer see the reviewer field.

Note:
Once the module becomes more stable, update hooks will be used instead of uninstalling/installing the module.
